### PR TITLE
Add prefix to commands for easy discovery and Snowplow Logo (Close #18)

### LINKED ISSUE #18: Suggestion: prefix commands with `Snowboard`
The commands in the command palette can be a little difficult to discover if someone is not familiar with all the features of the package, and this will only get worse as more commands are added.

Many other packages use the extension name as a prefix for their commands to help keep their commands together and allow for the user to explore more commands they may not be aware of.

If you are happy with this suggestion I'm happy to make the basic changes needed, but I have not developed a vscode package before so am not sure if there would be further work beyond simply incrementing the version number in the `package.json` to increment and release the package.

### PR DESCRIPTION
**Pull request by rlh1994**
This PR prefixes the commands in the command palette for easier identification of which commands belong to this extension, also swaps the default logo out for the Snowplow logo.

I've tested it locally and it all seems to build and work, but I am not an extension developer so I'm not sure what else needs doing when it comes to version/building/releasing.

### CODE REVIEW

**Comment by jethron** [comment]
LGTM.

Docs say the icon should be 24x24 but since it's an SVG I don't think it matters much, it seems to scale down fine. 
image

Will probably hold off on a release until #20 lands.

We left the icon as default originally because we didn't have any copyright for the Snowplow logo. Is that all OK to switch over @miike, even if the publisher is still marked as Poplin Data?

**Review by jethron — APPROVED** [review]

**Comment by miike** [comment]
Yeah - we now have rights to use the mark so we should be good to go.

**Pull request closed without merging**
